src/libraries/Microsoft.Extensions.FileSystemGlobbing/ref/Microsoft.Extensions.FileSystemGlobbing.cs

```diff
@@ -247,14 +247,14 @@ public partial class PatternContextRaggedInclude : Microsoft.Extensions.FileSyst
         public override void Declare(System.Action<Microsoft.Extensions.FileSystemGlobbing.Internal.IPathSegment, bool> onDeclare) { }
         public override bool Test(Microsoft.Extensions.FileSystemGlobbing.Abstractions.DirectoryInfoBase directory) { throw null; }
     }
-    public abstract partial class PatternContext<TFrame> : Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext
+    public abstract partial class PatternContext<TFrame> : Microsoft.Extensions.FileSystemGlobbing.Internal.IPatternContext where TFrame : struct
```

Comment:
I believe this is a breaking change. The type is in an `Internal` namespace and I'm not sure others actually use this.

cc @terrajobst @adamsitnik @Jozkee - thoughts on whether this break is "worth it"?

Reply:
Welp, people is not supposed to use this type. As per the [docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.filesystemglobbing.internal.patterncontexts.patterncontext-1.pushdataframe?view=dotnet-plat-ext-6.0):
> This API supports infrastructure and is not intended to be used directly from your code. This API may change or be removed in future releases.

IMO is not worth correcting (file a breaking change doc for something that no one is supposed to use) "internal" APIs like this one.

Reply:
> (file a breaking change doc for something that no one is supposed to use)

OTOH, If this is not an inconvenient at all, we can do whatever we want to this API.

Reply:
Well the docs say "This API may change". So I'm OK with changing it. Are the area-owners OK with it?  @dotnet/area-extensions-filesystem ?

Reply:
I'm one owner and I'm OK with it :)